test/test_files/ddl/sequence.test

```diff
@@ -17,10 +17,10 @@ Binder exception: test already exists in catalog.
 1
 -STATEMENT DROP TABLE test;
 ---- 1
-Table test has been dropped.
+Table test dropped.
```

Comment:
hmm is this grammatically complete? I would revert it back to `Table test has been dropped.`.